src/styles/index.scss

```diff
@@ -1,3 +1,112 @@
+$link-color: #00acdc;
+$text-regular-color: #616070;
+$text-bold-color: #060b35;
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
```

Comment:
Put each BEM block into separate file with the corresponding name.
`.card -> _card.scss`
`.stars -> _stars.scss`